Cargo.toml

```diff
@@ -196,161 +196,161 @@ rust-version = "1.83.0"
 
 [workspace.dependencies]
 aead = { version = "0.5", default-features = false }
-aes = "0.8.4"
-aes-gcm = "0.10.3"
-aligned-cmov = "2.3"
-anyhow = "1.0.80"
-assert_cmd = "2.0.14"
-assert_matches = "1.5.0"
+aes = "0.8"
+aes-gcm = "0.10"
+aligned-cmov = "2"
+anyhow = "1"
+assert_cmd = "2"
+assert_matches = "1"
 async-channel = "2"
 backtrace = "0.3"
-balanced-tree-index = "2.3"
+balanced-tree-index = "2"
 base64 = { version = "0.21", default-features = false }
-bincode = "1.3"
-bitflags = { version = "2.4", default-features = false }
-blake2 = { version = "0.10.6", default-features = false }
-bs58 = "0.4.0"
+bincode = "1"
+bitflags = { version = "2", default-features = false }
+blake2 = { version = "0.10", default-features = false }
+bs58 = "0.4"
 bulletproofs-og = { version = "3.0.0-pre.1", default-features = false }
```

Comment:
`-pre` versions receive special treatment and cargo requires that they are explicitly specified. This dependency is overriden in the `[patch.crates-io]` section at the bottom to point to a specific commit of this specific version.